### ISSUE TITLE
[v2.14.2] Ctrl+Left/Right adopts old behavior when an .inputrc file is present

### ISSUE THREAD

**Issue by ChuangTseu** [git-for-windows/git#1310]
- [x] I was not able to find an [open](https://github.com/git-for-windows/git/issues?q=is%3Aopen) or [closed](https://github.com/git-for-windows/git/issues?q=is%3Aclosed) issue matching what I'm seeing

### Setup

 - Which version of Git for Windows are you using? Is it 32-bit or 64-bit?

```
$ git --version --build-options

git version 2.14.2.windows.1
built from commit: 788debdfb6b794425a3b949dd1a0eb2425a72e96
sizeof-long: 4
machine: x86_64
```

 - Which version of Windows are you running? Vista, 7, 8, 10? Is it 32-bit or 64-bit?

```
$ cmd.exe /c ver

Microsoft Windows [Version 10.0.15063]
```

 - What options did you set as part of the installation? Or did you choose the
   defaults?

```
# One of the following:
> type "C:\Program Files\Git\etc\install-options.txt"
> type "C:\Program Files (x86)\Git\etc\install-options.txt"
> type "%USERPROFILE%\AppData\Local\Programs\Git\etc\install-options.txt"
$ cat /etc/install-options.txt

Path Option: Cmd
SSH Option: OpenSSH
CURL Option: OpenSSL
CRLF Option: CRLFAlways
Bash Terminal Option: MinTTY
Performance Tweaks FSCache: Enabled
Use Credential Manager: Enabled
Enable Symlinks: Enabled
```

 - Any other interesting things about your environment that might be related
   to the issue you're seeing?

I had an existing `.inputrc` file with the following content:
```
"\e[1;5D": backward-word
"\e[1;5C": forward-word`
```

### Details

 - Which terminal/shell are you running Git from? e.g Bash/CMD/PowerShell/other

Bash (mintty)

### The problem

Basically, I just removed the manual mappings of `;5C` and `;5D` from my `.inputrc` since those are no longer needed for jumping words with Ctrl+Left and Ctrl+Right with the latest update.

But it didn't work, I still had those dreaded `;5C;5D` when restarting bash.

Deleting the `.inputrc` file solved the problem, Ctrl+Left and Ctrl+Right now works like said in the changelog.

But it obviously isn't a good solution as some people might have other mappings they need to keep.

**Comment by elieux** [comment]
I don't think there's much to fix.  If you use a custom `.inputrc`, it will take precedence over the default one; that's just how *readline* is.  You can of course put `$include /etc/inputrc` in your custom file to have it load all the sweet defaults as well.
